README.md

```diff
@@ -154,6 +195,25 @@ be removed by the remove command without snapshot_lvm_verify_only.
 snapshot_lvm_verify_only is intended to be used to double check that the snapshot or
 remove command have completed the operation correctly.
 
+### snapshot_lvm_mountpoint_create
+
+If the mount point specified doesn't currently exist, create the mount point and any
+parent directories necessary for the mount point.
```

Comment:
When used inside of a snapset definition, use the `mountpoint_create` parameter.